doc/help.rst

```diff
@@ -2,38 +2,85 @@
 Help
 ####
 
-You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods.
+You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods. This allows you to access documentation easily directly from Spyder not having to break your workflow.
+
+|
+
+<div style="width:100%;height:0;padding-bottom:56%;position:relative;"><iframe src="https://giphy.com/embed/ghTnHFBPVB5P5kqEYz" width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div><p><a href="https://giphy.com/gifs/ghTnHFBPVB5P5kqEYz">via GIPHY</a></p>
```

Comment:
@juanis2112, you can't introduce html directly into Rest (contrary to Markdown). For this to work, you need to use the `raw` directive, as explained [here](https://groups.google.com/d/msg/sphinx-users/d6gg86LnItY/YZHQx_eXBmoJ).